pkg/codegen/go/gen_program.go

```diff
@@ -611,7 +611,7 @@ func GenerateProjectFiles(project workspace.Project, program *pcl.Program,
 		// pkg is the package name, we transformed these into Go paths above so use the map generated there
 		goPath, ok := packagePaths[pkg]
 		if ok {
-			err = gomod.AddReplace(goPath, "", path, "")
+			err = gomod.AddReplace(strings.TrimSuffix(goPath, "/"), "", path, "")
```

Comment:
Does this need TrimSuffix now that we've removed it from line 524?

Reply:
It can be removed ✅